sig-contributor-experience/role-handbooks/apac-coordinator.md

```diff
@@ -0,0 +1,25 @@
+# Contributor Experience APAC Coordinator
+
+## Overview
+
+The Kubernetes contributor base is still heavily skewed towards the US, and we are trying to diversify geographically. APAC Coordinator in Contributor Experience SIG helps to encourage and inspire more people in APAC regions to start contributing and ensure that there is adequate representation from there.
+
+Some common tasks include:
+
+- help with mirroring programs and strategies for contributors there
```

Comment:
nit: repetition of the word "there". Instead, maybe s/there/from APAC regions?

Reply:
how about [this](https://github.com/kubernetes/community/compare/720785e04a87ff3f43899ee7a08f6eb2d40616d8..572cbfe0e9f538cc4c356299b3a3e8e1be190496)?